test/parallel/test-fs-writev.js

```diff
@@ -0,0 +1,100 @@
+// Flags: --expose_externalize_string
+
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const path = require('path');
+const fs = require('fs');
+const tmpdir = require('../common/tmpdir');
+
+tmpdir.refresh();
+
+const expected = 'ümlaut. Лорем 運務ホソモ指及 आपको करने विकास 紙読決多密所 أضف';
+const getFileName = () =>
+  path.join(tmpdir.path,
+            `writev${Math.random().toString(36).substring(2, 15)}.txt`);
```

Comment:
Instead of using a random number, why not just use an incrementing counter to simplify things?

```js
{
  let i = 0;
  var getFileName = () => path.join(tmpdir.path, `writev${i++}.txt`)
}
```

Reply:
I have simplified it in another way, can you check it, please?